### PR TITLE
Bug fix: Enemy and hero dont cover grid but walls do

### PR DESCRIPTION
**Pull request by matharotheelf**

### CODE REVIEW

File: src/enemy.py [hunk 1]
```diff
@@ -10,10 +10,10 @@ class Enemy(pg.sprite.Sprite):
...
         self.game = game
-        self.image = pg.Surface((TILESIZE, TILESIZE))
+        self.image = pg.Surface((TILESIZE - 1, TILESIZE -1 ))
         self.image.fill(RED)
         self.rect = self.image.get_rect()
         self.x = x_pos
         self.y = y_pos
-        self.rect.x = TILESIZE * x_pos
-        self.rect.y = TILESIZE * y_pos
+        self.rect.x = TILESIZE * x_pos + 1
```
Comment (michaelnguyen974) [inline src/enemy.py:18]: nice addition of moving the enemy by 1 pixel

File: src/enemy.py [hunk 2]
```diff
@@ -10,10 +10,10 @@ class Enemy(pg.sprite.Sprite):
     def __init__(self, game, x_pos, y_pos):
         pg.sprite.Sprite.__init__(self)
         self.game = game
-        self.image = pg.Surface((TILESIZE, TILESIZE))
+        self.image = pg.Surface((TILESIZE - 1, TILESIZE -1 ))
```
Comment (michaelnguyen974) [inline src/enemy.py:13]: Nice work on changing the tile size by -1

**Review by michaelnguyen974 — APPROVED** [review]

**Review by CKKH — APPROVED** [review]
Lovely 🐛 fix, thanks Tom.

**Review by hotblack86 — APPROVED** [review]
Great spot and well fixed!

**Pull request merged**